MANIFEST.SKIP

```diff
@@ -41,5 +40,5 @@
 \B\._
 # Avoid archives of this distribution
 \bLINZ-Utils-[\d\.\_]+
-^MYMETA\.yml$
-^MYMETA\.json$
+^MYMETA\.yml$
+^MYMETA\.json$

```

Comment:
Why did these lines change?

Reply:
no clue

Reply:
Oh, it's just trailing whitespace. Never mind.